pytorch_lightning/metrics/functional/deprecated.py

```diff
@@ -0,0 +1,28 @@
+from pytorch_lightning.metrics.utils import to_categorical as new_to_categorical
+from pytorch_lightning.metrics.utils import to_onehot as new_to_onehot
+
+import warnings
+
+
+# TODO: remove in 1.1.1
```

Comment:
1.1.1 is the next release so it won't be for any release... so remove in 1.2?

Reply:
Will do